unikernel_single.ml

```diff
@@ -222,7 +222,7 @@ let unikernel_single_layout ~csrf unikernel now console_output =
                                                     text-primary-600 uppercase";
                                                  ];
                                              ]
-                                           [ txt "Host device" ];
+                                           [ txt "Unikernel block device" ];
```

Comment:
```suggestion
                                           [ txt "Block device" ];
```
I wonder if it's better to just say "Block device", since we already know the table contains unikernel information.